aiounifi/models/traffic_rule.py

```diff
@@ -0,0 +1,134 @@
+"""Traffic rules as part of a UniFi network."""
+
+from dataclasses import dataclass
+
+from typing_extensions import NotRequired, TypedDict
```

Comment:
The library expects python3.11 so you can import them from typing
```suggestion
from typing import NotRequired, TypedDict
```

Reply:
Already made this adjustments locally. Thanks for the feedback.